apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/ErrorReporting/ErrorReportingExceptionLoggerTest.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using System;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Diagnostics.AspNetCore.Tests.ErrorReporting
+{
+    public class ErrorReportingExceptionLoggerTest
+    {
+        private static readonly string _projectId = "pid";
```

Comment:
Make the constants const? Otherwise, make the static field names `s_projectId` etc. (I don't much mind which.)